### DISCUSSION TITLE
Feature request: Audit log/Perms to modify/delete users' own creations only

### DISCUSSION THREAD

**Discussion by joshbgosh10592** [OpenRCT2/OpenRCT2#15718] [Feature requests]
Hello!
I have a feature that I know I would love, and I believe a lot of admins would as well.
I'm requesting the ability to allow certain level users (for example, new users) to only be allowed to modify/delete their own creations. Then, higher level users (advanced builders/moderators/admins) to modify everything. Basically, a check box in the group permissions that says, "allow to modify own only" or something similar.
Then, because of this requiring an audit log for each ride, an overall audit log for the park would be awesome.

**Comment by Ryder17z** [comment]
I'd say it would also be useful if you could restrict the user so they can only modify a specific area

**Comment by IntelOrca** [comment]
There is no way of storing ownership of "things" without a new save format.

**Comment by joshbgosh10592** [comment]
I'd say both. But I know I like to intertwine coasters, so restricting users to areas is difficult. Also, that wouldn't really help with trolls (from what I can think of).
Just maybe an addition for when you have your own?

**Comment by Ryder17z** [comment]
At least you could confine the troll this way and keep other areas unmodifiable.

**Comment by fromthere** [comment]
How about adding multi meta file for save file? so sv6 format remains same, and we could save some orct data to seperate meta file.
